samcli/commands/docs/core/command.py

```diff
@@ -0,0 +1,151 @@
+"""
+Module contains classes for creating the docs command from click
+"""
+from typing import List, Optional
+
+from click import Command, Context, MultiCommand, style
+
+from samcli.cli.row_modifiers import RowDefinition
+from samcli.commands.docs.command_context import COMMAND_NAME, DocsCommandContext
+from samcli.commands.docs.core.formatter import DocsCommandHelpTextFormatter
+
+HELP_TEXT = "NEW! Open the documentation in a browser."
```

Comment:
shall we remove `NEW!` word after some time :)

Reply:
Yup, we'll remove it in a few months.